test/e2e/scheduling/preemption.go

```diff
@@ -186,10 +186,21 @@ var _ = SIGDescribe("SchedulerPreemption [Serial]", func() {
 				Limits:   podRes,
 			},
 		})
+		// Make sure that the lowest priority pod gets deleted
+		var podPreempted bool
+		// Wait till pod gets deleted
+		err := wait.PollImmediate(framework.Poll, framework.PodDeleteTimeout, func() (bool, error) {
+			preemptedPod, err := cs.CoreV1().Pods(pods[0].Namespace).Get(context.TODO(), pods[0].Name, metav1.GetOptions{})
+			if (err != nil && apierrors.IsNotFound(err)) || (err == nil && preemptedPod.DeletionTimestamp != nil) {
+				podPreempted = true
+				return podPreempted, nil
+			} else {
```

Comment:
sorry, based on golint you can drop the else here and just return false,err outside the if

Reply:
same on the other one